CONTRIBUTING.md

```diff
@@ -20,17 +20,11 @@ Type must be one of the following:
 * **test**: Adding missing tests or correcting existing tests
 * **revert**: A commit revert
 
-Scope is typically the most significant Go package that was changed, or the main component. Typical scopes:
+The scope must be [HMSPROV](https://issues.redhat.com/projects/HMSPROV) Jira issue.
 
-* dao
-* cache
-* clients
-* oapi
-* readme
-* changelog
-* scripts
-
-For WIP branches, start commit message with `WIP` and it will pass on CI.
+For **feat** and **fix** types a Jira issue link is required.
+Please use the scope like **feat(HMSPROV-XXX): subject**
+or put the issue reference in commit body as `Fixes: HMSPROV-XXX` or `Refs: HMSPROV-XXX`
```

Comment:
So you want to use full HMSPROV- prefix? That's way to long, but I won't block this if others like it.

Remove the info about commit body, I believe this is wrong, the github log Python library is not cabable scanning anything other than the first line, is it?

Reply:
It is and it even works with the current template we are using it works for GH issues the same way and I kinda like it, you want to link a ticket, but in the subject it doesn't tell you much usually, so having it in subject is not best IMHO.

See the changes in CHANGELOG.md in this patch and the relevant commits, most of them have the Jira issue linked in body, not subject.